src/templates/admin/elements/forms/required_field_info.html

```diff
@@ -0,0 +1,6 @@
+{% load i18n %}
+<p>
+    {% trans "Required fields are marked with" %} 
+    <abbr class="red req" title='{% trans "required" %}'>*</abbr>
+    .
```

Comment:
Can we put the full stop on the same line so there's no white space after the asterisk?

<img width="292" height="59" alt="image" src="https://github.com/user-attachments/assets/29475568-686c-44c4-9616-a41e7219c7ad" />